valhalla/meili/state.h

```diff
@@ -53,7 +53,11 @@ class State {
       ++dest;
     }
     labelset_ = labelset;
+#ifdef LOGGING_LEVEL_TRACE
     LOG_TRACE("Found " + std::to_string(found) + " destinations out of " + std::to_string(dest - 1));
+#else
+    (void)found;
+#endif
```

Comment:
instead of this i would use the `[[maybe_unused]]` qualifier on the declaration of found. we are on c++17 now so we can totally use that feature

Reply:
Thanks for the tip! I hadn't heard of this functionality before.

I made this change in 51d23a74c098612202f345a7f071e0c7d2ce110b